app/views/exports/_notification_pane.html.haml

```diff
@@ -18,4 +18,6 @@
     %a.btn.btn-primary{ href: '#export-options', data: { toggle: "tab" } }
       %i.fa.fa-arrow-left
       Back
-    = f.submit 'Export', class: 'btn btn-primary pull-right'
\ No newline at end of file
+    = f.submit 'Export', class: 'btn btn-primary pull-right'
```

Comment:
we should get rid of all the `.pull-right` and revamp these buttons to just be next to each other aligned to the right using `.text-right` on the section above.  This will make it more consistent with other button changes I've outlined below.